engine/src/main/scala/cromwell/engine/backend/BackendCall.scala

```diff
@@ -136,11 +136,6 @@ trait BackendCall {
 
   def useCachedCall(cachedBackendCall: BackendCall)(implicit ec: ExecutionContext): Future[ExecutionHandle] = jobDescriptor.useCachedCall(cachedBackendCall.jobDescriptor)
 
-  /**
-    * Return CallLogs which contains the stdout/stderr of the particular call
-    */
-  def stdoutStderr: CallLogs
-
```

Comment:
It might not be that bad as this removes things from the BackendCall, which is going in the right direction since this file doesn't exist anymore in my branch.